graph-db/serverless/src/utils/cmr/getEchoToken.js

```diff
@@ -9,7 +9,12 @@ export const getEchoToken = async () => {
   const { env: { IS_LOCAL } } = process
 
   if (IS_LOCAL === 'true') {
-    return null
+    let localToken = process.env.TOKEN
```

Comment:
Does this work? The other `process.env` variables in this file have extra setup in `serverless.yml` you are missing

Reply:
yes, this is a setting that is only needed when running bootstrap locally i.e. when `IS_LOCAL === 'true'`. We don't need this configuration when deployed in AWS.